cmd/cloud/deployment_workerqueue.go

```diff
@@ -45,7 +45,7 @@ func newDeploymentWorkerQueueCreateCmd(out io.Writer) *cobra.Command {
 	cmd.Flags().StringVarP(&deploymentID, "deployment-id", "d", "", "The deployment where the worker queue should be deleted.")
 	cmd.Flags().StringVarP(&deploymentName, "deployment-name", "", "", "Name of the deployment where the worker queue should be deleted.")
 	cmd.Flags().StringVarP(&name, "name", "n", "", "The name of the worker queue. Queue names must not exceed 63 characters and contain only lowercase alphanumeric characters or '-' and start with an alphabetical character.")
-	cmd.Flags().IntVarP(&minWorkerCount, "min-count", "", 0, "The min worker count of the worker queue.")
+	cmd.Flags().IntVarP(&minWorkerCount, "min-count", "", -1, "The min worker count of the worker queue.")
 	cmd.Flags().IntVarP(&maxWorkerCount, "max-count", "", 0, "The max worker count of the worker queue.")
```

Comment:
nit: just checking whether we won't run into similar problems with max-count default as zero, or it's not so relevant from a functional pov?

Reply:
we are focusing on `min-count` only as that is allowed to be 0. Api floors for `max-count` and `concurrency` is 1 so it can not be 0.